src/coord/src/catalog/builtin.rs

```diff
@@ -1557,18 +1557,46 @@ FROM mz_catalog.mz_schemas",
 pub const PG_CLASS: BuiltinView = BuiltinView {
     name: "pg_class",
     schema: PG_CATALOG_SCHEMA,
+    /*
+     * MZ doesn't support typed tables so reloftype is filled with 0
```

Comment:
Can you move these comments inline with the query, i.e. to the appropriate line ? prefixing a line with `--` makes it a comment

Reply:
Moved